app/views/posts/new.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="row">
+    <div class="col-sm-3"></div> 
+    <div class="col-sm-6">
+      <h1>New Post</h1>
```

Comment:
[OPTIONAL] 

If in the future you find yourself in need of having this form in another view, e.g. home view and user profile view, then it could be idea to have this in a partial.